Makefile

```diff
@@ -144,7 +115,7 @@ go-vet:
 	go vet ./...
 
 .PHONY: go-lint
-go-lint:
+go-lint: cmd/cli/chart.tgz pkg/envoy/lds/stats.wasm
```

Comment:
Why does go-lint depend on the chart and wasm module? This is a static analyzer so wondering why the dependency.